Recrutify/Recrutify.Host/Controllers/ProjectsController.cs

```diff
@@ -84,13 +85,15 @@ public async Task<ActionResult> DeleteAsync([FromRoute] Guid id, CancellationTok
         [HttpGet("{id:guid}")]
         public async Task<ActionResult<ProjectDTO>> GetByIdAsync([FromRoute] Guid id)
         {
-            var project = await _projectService.GetAsync(id);
-            if (project == null)
+            try
+            {
+                var project = await _projectService.GetAsync(id);
+                return Ok(project);
+            }
+            catch (NotFoundException)
```

Comment:
You should catch not only here, but in all places where you call project service get method - in all controllers where this exception can be thrown

Reply:
that's all